release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3158,6 +3158,35 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-14-9"]
+=== RHSA-2024:0204 - {product-title} 4.14.9 bug fix and security update
+
+Issued: 2024-01-17
+
+{product-title} release 4.14.9, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0204[RHSA-2024:0204] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:0207[RHSA-2024:0207] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.9 --pullspecs
+----
+
+[id="ocp-4-14-9-bug-fixes"]
+==== Bug fixes
+
+* Previously,  The Cluster Version Operator (CVO) continually retrieves update recommendations and evaluates known conditional update risks against the current cluster state. CVO changes caused failing risk evaluations to block the CVO from fetching new update recommendations. This bug caused the CVO to fail to notice the update recommendation service serving an improved risk declaration. With this release, the CVO continues to poll the update recommendation service, regardless of whether update risks are being successfully evaluated or not. (link:https://issues.redhat.com/browse/OCPBUGS-26207[*OCPBUGS-26207*])
+
+* Previously, The use of eus-* channels for mirroring releases was causing a failure in mirroring with oc-mirror. This was due to the fact that oc-mirror did not acknowledge that eus-* channels are even-numbered only.
```

Comment:
One thing I thought of post-review, should oc-mirror be oc-mirror plugin? 🤔  #notmyprodarea so idk just searching in the OCP docs repo made me think it may need some clarification.

Reply:
That might make sense, I'll ask the dev.

Reply:
Good call, it is the plugin.